src/mito2/src/sst/parquet/format.rs

```diff
@@ -88,6 +90,7 @@ impl WriteFormat {
             self.arrow_schema.fields().len()
         );
         let mut columns = Vec::with_capacity(batch.fields().len() + FIXED_POS_COLUMN_NUM);
+        let mut null_buf: HashMap<ConcreteDataType, ArrayRef> = HashMap::new();
```

Comment:
Could we add a `null_array_cache: HashMap<ConcreteDataType, ArrayRef>` as a field in the `WriteFormat`? Then we can reuse null arrays across batches.